src/routes/RoutesContainer.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { Home } from '../pages';
+import Mockman from 'mockman-js';
+
+const RoutesContainer = () => {
+  return (
+    <Routes>
+      <Route path='/' element={<Home />} />
+      <Route path='/home' element={<Home />} />
+      <Route path='/mock' element={<Mockman />} />
+    </Routes>
+  );
+};
```

Comment:
suggestion: use useRoutes() hook instead

Reply:
thanks for the suggestion